database/console/seed_command.go

```
@@ -0,0 +1,113 @@
+	"github.com/goravel/framework/contracts/console/command"
+	"github.com/goravel/framework/contracts/database/seeder"
+	"github.com/goravel/framework/contracts/foundation"
+)
+
+type SeedCommand struct {
+	config config.Config
+	app    foundation.Application
+}
+
+func NewSeedCommand(config config.Config, app foundation.Application) *SeedCommand {
+	return &SeedCommand{
+		config: config,
+		app:    app,
+	}
+}
+
+// Signature The name and signature of the console command.
+func (receiver *SeedCommand) Signature() string {
+	return "db:seed"
+}
+
+// Description The console command description.
+func (receiver *SeedCommand) Description() string {
+	return "Seed the database with records"
+}
+
+// Extend The console command extend.
+func (receiver *SeedCommand) Extend() command.Extend {
+	return command.Extend{
+		Category: "db",
+		Flags: []command.Flag{
+			&command.BoolFlag{
+				Name:    "force",
+				Aliases: []string{"f"},
+				Usage:   "force the operation to run when in production",
+			},
+			&command.StringFlag{
+				Name:    "seeder",
+				Value:   "",
+				Aliases: []string{"s"},
+				Usage:   "name of the seeder to run",
+			},
+		},
+	}
+}
+
+// Handle executes the console command.
+func (receiver *SeedCommand) Handle(ctx console.Context) error {
+	if !receiver.ConfirmToProceed(ctx) {
+		return nil
+	}
+
+	color.Greenln("Seeding database.")
+	seeder := receiver.GetSeeder(ctx)
+	if seeder == nil {
+		log.Println("No valid seeder instance found.")
+		return nil
+	}
+	seeder.Run()
+	return nil
+}
+
+// ConfirmToProceed determines if the command should proceed based on user confirmation.
+func (receiver *SeedCommand) ConfirmToProceed(ctx console.Context) bool {
+	force := ctx.OptionBool("force")
+	if force || (receiver.config.Env("APP_ENV") != "production") {
+		return true
+	}
+
+	// Display production alert message
+	receiver.config.Add("alert", "Application In Production")
+
+	alert := receiver.config.GetString("alert")
+
+	color.Yellowln(alert)
```

S: it's better to return an error and print the error in the `Handle` method.